tests/scripts/faketime-all.sh

```diff
@@ -380,6 +381,22 @@ check_tools()
     done
 }
 
+check_faketime() {
+
+    for i in /usr/local/lib/faketime/libfaketime.so.1 \
```

Comment:
Why not use the `faketime` program?

Reply:
It might cause make fail or other fail due to time stamp.
If we do not wrap all commands, make or other program might report fail. And I think this change is the easiest way to avoid those failure